github3/repos/issue_import.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from ..models import GitHubCore
+"""
+github3.repos.issue_import
+==========================
+
+This module contains the ImportedIssue object for Github's import issue API
+
+"""
+
+
+class ImportedIssue(GitHubCore):
+    """
+    The :class:`ImportedIssue <ImportedIssue>` object. This represents
+    information from the Import Issue API.
+
+    See also: https://gist.github.com/jonmagic/5282384165e0f86ef105
```

Comment:
@sigmavirus24 should I remove the reference to the gist here and into 'import_issue'? Or have both files contain the link?